service/store/src/main/resources/application.yml

```
@@ -22,4 +22,9 @@ spring:
 eureka:
   client:
     service-url:
-      defaultZone: ${EUREKA_SERVER_URL}
\ No newline at end of file
+      defaultZone: ${EUREKA_SERVER_URL}
+
+
+naver:
+  client-id:
+  client-secret: 
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)